app/controllers/openid_connect/logout_controller.rb

```diff
@@ -25,6 +31,15 @@ def logout
       end
     end
 
+    # +POST+ Handle logout request (with confirmation if initiated by relying partner)
+    # @see {OpenID Connect RP-Initiated Logout 1.0 Specification}[https://openid.net/specs/openid-connect-rpinitiated-1_0.html#RPLogout] # rubocop:disable Layout/LineLength
+    def create
+      flash[:event] = {
+        original_method: request.method.to_s,
+      }
```

Comment:
I think we'd prefer to write data to directly to the `session` instead of the flash, we currently use the flash for user-facing messages so we have code that will auto-render all messages stored there: https://github.com/18F/identity-idp/blob/47b139d732d83d6b60c7974dad35f23243bb3283/app/components/flash_component.rb#L12-L29

I think even if brief we wouldn't want to risk showing something?

Reply:
I guess using `:event` will be outside `VALID_FLASH_TYPES` but I still feel like direct session data is prob easier than flash

Reply:
Ok, will change.

Reply:
Using session now.